behavior/steps/gw/commonSteps/LoginSteps.ts

```diff
@@ -1,13 +1,29 @@
 const { Given } = require("@cucumber/cucumber")
 import { onApplication } from '@gtui/gt-ui-framework/bin/Utilities/OOTBSteps/CommonSteps/ISBaseStep';
-import { onApp } from '../../../../ui/pages/gw/registry/onApp'
+import { onApp } from "../../../../ui/pages/gw/registry/onApp"
```

Comment:
can we use double quotes as a standard for all imports?

Reply:
updated to single quotes as GT UI framework used same